duckduckgo/iOS
# don't extend edges under the top and bottom bars for the container view

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/416322909657029
CC:

**Description**:

The web view was hiding content expecting to be at the bottom of the page.  Solution was to not extend the container view under the top and bottom bars.

**Steps to test this PR**:
1. Go to donttrack.us - observe the "down" circle button at the bottom of the page.
2. Go to squarespace.com - after dismissing the cookie alert observe the "get started" button at the bottom of the page
3. Go to hugeinc.com - observe the text attached to the bottom of the screen, e.g. "Magenta\nRead our latest articles on design and technology."


###### Reviewer Checklist:


###### PR DRI Checklist:
